domain-mysql/src/main/java/com/kernelsquare/domainmysql/domain/notice/entity/Notice.java

```diff
@@ -0,0 +1,90 @@
+package com.kernelsquare.domainmysql.domain.notice.entity;
+
+import org.apache.commons.lang3.StringUtils;
+import org.hibernate.annotations.DynamicUpdate;
+
+import com.kernelsquare.core.common_response.error.exception.InvalidParamException;
+import com.kernelsquare.core.util.TokenGenerator;
+import com.kernelsquare.domainmysql.domain.base.BaseEntity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+
+@Entity(name = "Notice")
+@Table(name = "notice")
+@Getter
+@DynamicUpdate
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Notice extends BaseEntity {
+	private final String NOTICE_PREFIX = "ntc_";
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@Column(nullable = false, unique = true, name = "notice_token", columnDefinition = "varchar(50)")
+	private String noticeToken;
+
+	@Column(nullable = false, name = "notice_title", columnDefinition = "varchar(50)")
+	private String noticeTitle;
+
+	@Column(nullable = false, name = "notice_content", columnDefinition = "text")
+	private String noticeContent;
+
+	@Enumerated(EnumType.STRING)
+	@Column(nullable = false, name = "notice_category", columnDefinition = "varchar(20)")
+	private NoticeCategory noticeCategory;
+
+	@Getter
+	@RequiredArgsConstructor
+	public enum NoticeCategory {
+		GENERAL("일반 공지"),
+		QNA("Q&A 관련 공지"),
+		COFFEE_CHAT("커피챗 관련 공지");
+
+		private final String description;
+	}
+
+	@Builder
+	public Notice(String noticeTitle, String noticeContent, NoticeCategory noticeCategory) {
+		if (StringUtils.isBlank(noticeTitle))
+			throw new InvalidParamException("Notice.noticeTitle");
+		if (StringUtils.isBlank(noticeContent))
+			throw new InvalidParamException("Notice.noticeContent");
+		if (noticeCategory == null)
```

Comment:
확인해보니 Objects.isNull 내부 구현이 아래처럼 되어있네요!
```java
    /**
     * Returns {@code true} if the provided reference is {@code null} otherwise
     * returns {@code false}.
     *
     * @apiNote This method exists to be used as a
     * {@link java.util.function.Predicate}, {@code filter(Objects::isNull)}
     *
     * @param obj a reference to be checked against {@code null}
     * @return {@code true} if the provided reference is {@code null} otherwise
     * {@code false}
     *
     * @see java.util.function.Predicate
     * @since 1.8
     */
    public static boolean isNull(Object obj) {
        return obj == null;
    }
```

```java
    if (StringUtils.isBlank(noticeContent))
	throw new InvalidParamException("Notice.noticeContent");
   if (Objects.isNull(noticeCategory))
	throw new InvalidParamException("Notice.noticeCategory");
```

위와 같은 방식으로 바로 윗 라인인과 통일하는 편이 가독성 측면에서 좋을거 같아서 Objects.isNull로 수정하고자 하는데 어떻게 생각하시나요?